app/templates/administrative-units/administrative-unit/governing-bodies/governing-body/mandatory/edit.hbs

```diff
@@ -0,0 +1,278 @@
+<div class="au-c-body-container au-c-body-container--scroll">
+  <div class="au-o-box au-o-flow au-o-flow--large">
+    <div class="au-o-flow au-o-flow--tiny">
+      <AuHeading>
+        Bewerk mandataris
+      </AuHeading>
+
+      <AuHeading @level="2" @skin="3" class="au-u-margin-top-tiny">
+        {{@model.person.givenName}}
+        {{@model.person.familyName}}
+      </AuHeading>
+    </div>
+
+    <DataCard>
+      <:title>Persoonlijke gegevens</:title>
+      <:card as |Card|>
+        <Card.Columns>
+          <:left as |Item|>
+            <Item>
+              <:label>Naam</:label>
+              <:content>
+                {{@model.person.givenName}}
+                {{@model.person.familyName}}
+              </:content>
+            </Item>
+          </:left>
+
+        </Card.Columns>
+      </:card>
+    </DataCard>
+    <form
+      id="edit-mandatory-form"
+      aria-label="edit-mandatory-form"
+      {{on "submit" (perform this.save)}}
+    >
+      <div class="au-o-flow au-o-flow--large">
+        <EditCard>
+          <:title>Positie</:title>
+          <:card as |Card|>
+            <Card.Columns>
+              <:left as |Item|>
+                <Item>
+                  <:label>Type</:label>
+                  <:content>
+                    <AuInput
+                      @width="block"
+                      @value="Mandaat"
+                      disabled={{true}}
+                    />
+                  </:content>
+                </Item>
+                <Item>
+                  <:label>Bestuursfunctie</:label>
+                  <:content>
+                    <AuInput
+                      @width="block"
+                      @value={{this.mandatory.mandate.roleBoard.label}}
+                      disabled={{true}}
+                    />
+                  </:content>
+                </Item>
+              </:left>
+              <:right as |Item|>
+                <Item>
+                  <:label>Naam Eredienst</:label>
+                  <:content>
+                    <AuInput
+                      @width="block"
+                      @value={{this.administrativeUnit.name}}
+                      disabled={{true}}
+                    />
+                  </:content>
+                </Item>
+              </:right>
+            </Card.Columns>
+          </:card>
+        </EditCard>
+
+        <EditCard>
+          <:title>Primaire contactgegevens</:title>
+          <:card as |Card|>
+            <Card.Columns>
+              <:left as |Item|>
+                <Item @labelFor="contact-email">
```

Comment:
I moved this to the right column since there are a lot of address fields already. Or do you think this version is better?

Reply:
It makes sense.